run-tests.php

```diff
@@ -1889,6 +1892,8 @@ function run_test(string $php, $file, array $env): string
     $org_file = $file;
     $orig_php = $php;
 
+    $php = escapeshellarg($php);
```

Comment:
might be hard to read, better escape individually

Reply:
The $php variable is updated anyway later on in the script. So I think this is okay tbh.